logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/conversation/message/MLSMessageFailureHandler.kt

```
@@ -50,6 +50,9 @@ internal object MLSMessageFailureHandler {
             is MLSFailure.InternalErrors,
             is MLSFailure.Disabled,
             MLSFailure.CommitForMissingProposal,
+            MLSFailure.BufferedCommit, // TODO should be ignored?
+            is MLSFailure.MessageRejected, // TODO should be ignored?
+            MLSFailure.OrphanWelcome, // TODO should be ignored?
```

@typfel should those errors be ignored or not ?